### PR TITLE
minor: update Orekit and Hipparcus checkout commits

### PR DESCRIPTION
**Pull request by nmancus1**
Let me know if you want me to squash this, but I figured I should add the directory removal as a separate commit.

### CODE REVIEW

**Comment by nmancus1** [comment]
PR to fix Travis here: #7794

File: .ci/wercker.sh [hunk 1]
```diff
@@ -75,9 +75,10 @@ no-error-orekit)
   # no CI is enforced in project, so to make our build stable we should
   # checkout to latest release/development (annotated tag or hash) or sha that have fix we need
   # git checkout $(git describe --abbrev=0 --tags)
-  git checkout "a7e67ce73803c67a""ad90e0b28ed77a7781dc28a9"
+  git checkout "b67b419db7014f4b""ad921a1bac6c848384ad2b92"
```
Comment (romani) [inline .ci/wercker.sh]: There is spelling false positive in Travis, to fix it we need split this Sha one more time, `bac` should be in different string values

Comment (nmancus1) [reply .ci/wercker.sh]: Oh, I see! I was wondering why the strings were broken up. I'll fix this right away.

**Comment by nmancus1** [comment]
> Next time it will be easier to provide such commits in same PR where this changes required.
Ok, I will make sure to do that in the future.  Thanks for explaining everything so well, and your understanding.

**Pull request closed without merging**